tests/platform_tests/thermal_control_test_helper.py

```diff
@@ -303,9 +303,10 @@ def __enter__(self):
         thermal control daemon to make it effect.
         :return:
         """
-        self.dut.command('mv -f {} {}'.format(self.thermal_policy_file_path, self.thermal_policy_file_backup_path))
-        self.dut.copy(src=os.path.join(FILES_DIR, self.src), dest=self.thermal_policy_file_path)
-        restart_thermal_control_daemon(self.dut)
+        if os.path.exists(self.thermal_policy_file_path):
+            self.dut.command('mv -f {} {}'.format(self.thermal_policy_file_path, self.thermal_policy_file_backup_path))
+            self.dut.copy(src=os.path.join(FILES_DIR, self.src), dest=self.thermal_policy_file_path)
```

Comment:
if there is no thermal_policy file on the device, are we expecting to run the test with a give file? or should the test be stopped or skipped?

Reply:
the testcase would handle missing thermal_policy gracefully - either skip or fail. The idea is that common class or fixtures should not be throwing these errors, since they get used in multiple testcases. In this case, although the device checker testcase had a skip, the error was seen before due to this failure

Reply:
So if the thermal policy is not in the dut, will the test be skipped? or is it supposed to test with the test policy file? The reason I asked was the 2nd line of the change which copy the src to the thermal policy file can be executed if the test is supposed to run the test policy file.

Reply:
Updated the files based on the discussion